pygenesys/tests/test_pygenesys.py

```diff
@@ -1,14 +1,29 @@
 from pygenesys import technology
+from pygenesys import driver
 
 
-def test_technology_placeholder():
-    """
-    Tests the placeholder function in
-    pygenesys.technology
-    """
-
+def test_technology_placeholder_is_none():
     ret_value = technology.placeholder()
 
     assert(ret_value is None)
 
     return
+
+
+def test_name_from_path():
+    assert(driver.name_from_path("/Users/test/test.py") == "test")
```

Comment:
You don't need to add a docstring for tests, but I think it's a good practice to add in a multiline comment describing the purpose of the test.